### PR TITLE
feat: ✨ Saving custom title

### PR DESCRIPTION
**Pull request by slugb0t**
## Summary by Sourcery

Add functionality to save a custom license title and enhance the license saving process by checking for changes before allowing a push to the repository.

New Features:
- Introduce the ability to save a custom license title when editing a license.

Enhancements:
- Add a check to prevent saving and pushing a license if no changes have been made to the original content.

### CODE REVIEW

File: ui/pages/dashboard/[owner]/[repo]/edit/license.vue [hunk 1]
```diff
@@ -363,19 +365,36 @@ const navigateToPR = () => {
...
+          <template #icon>
+            <Icon name="material-symbols:save" />
+          </template>
+          Save License Title
+        </n-button>
+  
+        <n-button
+          size="large"
+          color="black"
+          x
```
Comment (megasanjay) [inline ui/pages/dashboard/[owner]/[repo]/edit/license.vue:386]: Typo?

File: ui/pages/dashboard/[owner]/[repo]/edit/license.vue [hunk 2]
```diff
@@ -363,19 +365,36 @@ const navigateToPR = () => {
...
+          @click="saveLicenseAndPush"
+          :disabled="!licenseId || !licenseContent"
+          :loading="submitLoading"
+          v-if="licenseContent !== originalLicenseContent"
```
Comment (megasanjay) [inline ui/pages/dashboard/[owner]/[repo]/edit/license.vue:390]: I think this should always be shown

**Review by megasanjay — CHANGES_REQUESTED** [review]

File: ui/pages/dashboard/[owner]/[repo]/edit/license.vue (hunk 2 above)
Comment (slugb0t) [reply ui/pages/dashboard/[owner]/[repo]/edit/license.vue]: Gotcha, I'll remove the condition

File: ui/pages/dashboard/[owner]/[repo]/edit/license.vue [hunk 3]
```diff
@@ -363,19 +365,36 @@ const navigateToPR = () => {
...
-        @click="saveLicenseAndPush"
-        :disabled="!licenseId || !licenseContent"
-        :loading="submitLoading"
-      >
-        <template #icon>
-          <Icon name="ion:push" />
-        </template>
-        Save and push license to repository
-      </n-button>
+      <n-flex class="justify-between">
```
[the remark this answers is not in the document]
Comment (slugb0t) [reply ui/pages/dashboard/[owner]/[repo]/edit/license.vue]: Replaced with naive ui properties over tailwind styling

**Pull request merged**